docs/simulation/calculations.yaml

```diff
@@ -36,9 +32,34 @@ POPs:
     population_growth: Sum of population_growth modifiers
     pop_growth: Sum of pop_growth modifiers
     global_population_growth/10: Sum of global_population_growth modifiers
-  Consciousness from literacy:
-    formula: defines.pops.CON_LITERACY * Plurality * POP literacy * (1 + literacy_con_impact)
-  Life needs scalar:
-    formula: (1 + Plurality) * (1 + 2 * Consciousness / defines.pops.PDEF_BASE_CON) * (1 + sum of goods_demand) * defines.pop.BASE_GOODS_DEMAND * POP size / 200000
-  Everyday & luxury needs scalar:
-    formula: Life needs scalar * (1 + number of inventions * defines.pops.INVENTION_IMPACT_ON_DEMAND)
\ No newline at end of file
+  Attitudes:
+    Consciousness from literacy: defines.pops.CON_LITERACY * Plurality * POP literacy * (1 + literacy_con_impact)
+  Needs:
+    Life needs scalar: (1 + Plurality) * (1 + 2 * Consciousness / defines.pops.PDEF_BASE_CON) * (1 + sum of goods_demand) * defines.pop.BASE_GOODS_DEMAND * POP size / 200000
+    Everyday & luxury needs scalar: Life needs scalar * (1 + number of inventions * defines.pops.INVENTION_IMPACT_ON_DEMAND)
+  POP income:
+    Pensions: 2 * pension_level * administrative efficiency * social spending slider * base life needs price * (1 - effective tax) * POP size / 200000
+    Unemployment subsidies: 2 * unemployment_benefit * administrative efficiency * social spending slider * base life needs price * (1 - effective tax) * POP size / 200000
+    Wages:
+      Minimum (worker) wage: 2 * minimum_wage * administrative efficiency * base life needs price * (1 - effective tax) * POP size / 200000
+      RGO wages:
+        Owners:
+          if minimum worker wages > normal worker wage:
+            formula: (RGO income - total worker income) * (1 - effective tax)
+          else:
+            formula: RGO income * min(0.5, 2 * owners / workers) * owner POP size / owners * (1 - effective tax) 
+            owners: total size of owner POPs in state
+            workers: number of employed workers in RGO
+            min(0.5: Hardcoded maximum of half the RGO income.
+            2*: Hardcoded value.
+        Workers:
+          if is_slave: 0
+          else:
+            if minimum wage > normal wage:
+              formula: max(minimum wage, RGO income * employed workers in POP / total non-slave employed workers) * (1 - effective tax) 
```

Comment:
If there are no slaves workers, since each worker pop gets a share of the total `RGO income` proportional to their size (or more if `minimum wage` is greater), won't `total worker income` on line 48 always be `>= RGO income`, and so owner wages will always be `<= 0` when `minimum worker wages > normal worker wage`? (This is still the case with a `owner POP size / owners` added like I mention in my comment on line 48.)

Reply:
Minimum wage does not depend on RGO income.
Normal worker wage is a fraction of the RGO income.

"won't total worker income on line 48 always be >= RGO income [?]"
No.
If the RGO is profitable, the normal wage can be greater than the minimum wage.
Also total worker wage is capped at RGO income.
Even if minimum wage > normal wage, there can be plenty of profit left for the owners.

Example (no taxes):
RGO income = 200
employed workers (1 pop) = 10000
owners in state = 100

Normal total worker wage = 200*(1-2*100/10000) = 196
So per individual = 0.0196
Normal total owner wage = 200*2*100/10000 = 4

If minimum wage per individual ended up being 0.0198
Then each worker would get that and
Total owner wage = (200 - 10000 * 0.0198) = 2

So even with minimum wage, the owners can have a wage > 0.

"and so owner wages will always be <= 0"
Owner wages can never go negative.

Reply:
My concern was the second argument in the `max` function:
```
max(minimum wage, RGO income * employed workers in POP / total non-slave employed workers)
```
If we're in the one worker scenario, `employed workers in POP / total non-slave employed workers = 1`, so this becomes:
```
max(minimum wage, RGO income)
``
If `RGO income` is greater than minimum wage, then the worker gets the whole `RGO income` as its wages, or am I missing something?

Reply:
Thanks for pointing that out. The max should be min. So either minimum wage or a proportional share of the RGO income. This is what I meant with "Also total worker wage is capped at RGO income.".

Reply:
Ah ok, so it's still possible for an owner to get 0 wages, but only if minimum wages are so high that there's non left for him, which makes sense.